drivers/storage/portworx/component/monitoring.go

```diff
@@ -184,6 +186,7 @@ func (c *monitoring) retryCreate(
 func serviceMonitorLabels() map[string]string {
 	return map[string]string{
 		"name": PxServiceMonitor,
+		"app":  PxBackupServiceMonitor,
```

Comment:
These labels are added on the portworx service montior as well. Can you separate out this method into two, one for prometheus's service monitor selector labels and other for the labels on the service monitor itself?

Reply:
Added.